src/google/protobuf/compiler/objectivec/objectivec_generator.cc

```diff
@@ -89,6 +89,12 @@ bool ObjectiveCGenerator::GenerateAll(const std::vector<const FileDescriptor*>&
       // There is no validation that the prefixes are good prefixes, it is
       // assumed that they are when you create the file.
       generation_options.expected_prefixes_path = options[i].second;
+    } else if (options[i].first == "expected_prefixes_suppressions") {
+      // A semicolon delimited string that lists the paths of .proto files to
+      // skip the package prefix validations (expected_prefixes_path). This is
+      // provided as an "out", to skip some files being becked.
```

Comment:
What is becked?

A semicolon delimited string that lists the paths of .proto files to exclude from the package prefix validations...

Reply:
Done